src/main/kotlin/javabot/dao/impl/openweathermap/model/OWWeather.kt

```diff
@@ -0,0 +1,57 @@
+@file:Suppress("unused")
+
+package javabot.dao.impl.openweathermap.model
```

Comment:
data classes maybe?

Reply:
data classes work. I was using classes because I wasn't sure how morphia would treat the data classes, but everything seems fine.